e2e/test/E2EMsTestBase.cs

```diff
@@ -14,7 +14,7 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
 //Workers = 0 makes the test engine use one worker per available core. It does not mean to run serially.
-[assembly: Parallelize(Workers = 0, Scope = ExecutionScope.MethodLevel)]
+[assembly: Parallelize(Workers = 0, Scope = ExecutionScope.ClassLevel)]
```

Comment:
Want to try at this concurrency to see if the tests pass more frequently. We surely have concurrency issues.